lighthouse-extension/test/app/src/lighthouse-lr-background-test.js

```diff
@@ -6,12 +6,12 @@
 'use strict';
```

Comment:
@brendankenny is it ok if I renamed this file? we were testing LR anyway.

Reply:
>  is it ok if I renamed this file? we were testing LR anyway.

yep, makes sense